lib/profiles.js

```diff
@@ -413,7 +413,7 @@ export async function getAllPapersByGroupId(profileId, accessToken) {
   try {
     const notesList = await api.get(
       '/notes',
-      { 'content.authorids': profileId },
+      { 'content.authorids': profileId, invitation: 'dblp.org/-/record' },
```

Comment:
we will user another invitation for API 2 import, so we will need to change this later.

Reply:
yes it will be changed in the dblp import using v2 pr after this one is merged

Reply:
updated in https://github.com/openreview/openreview-web/pull/1716/commits/ab87d40cc59eaf86d0b91781a5e1a94ebd28756e